leonardo/dsvm-template.json

```diff
@@ -265,10 +265,7 @@
               }
             },
             "imageReference": {
-              "publisher": "microsoft-dsvm",
-              "offer": "ubuntu-1804",
-              "sku": "1804-gen2",
-              "version": "latest"
+              "id": "/subscriptions/3efc5bdf-be0e-44e7-b1d7-c08931e3c16c/resourceGroups/mrg-qi-1-preview-20210517084351/providers/Microsoft.Compute/galleries/msdsvm/images/customized_ms_dsvm/versions/0.0.4"
```

Comment:
this version should have all the packages required (other than tnu and fiss)...
it'll have jupyter login screen removed, but RStudio login screen is still there